Doctrine/Queue/SyncIndexWithObjectChangeProcessor.php

```diff
@@ -59,24 +59,28 @@ public function process(PsrMessage $message, PsrContext $context)
             return Result::reject('The message data misses id');
         }
         if (false == isset($data['index_name'])) {
-            return Result::reject('The message data misses id');
+            return Result::reject('The message data misses index_name');
         }
         if (false == isset($data['type_name'])) {
-            return Result::reject('The message data misses id');
+            return Result::reject('The message data misses type_name');
+        }
+        if (false == isset($data['repository_method'])) {
```

Comment:
this is a BC break, could we fall back to find method for now?

Reply:
The `find` is set as default [here](https://github.com/php-enqueue/enqueue-elastica-bundle/pull/14/files#diff-850942b3ba24ab03a40aaa81b6152852R38), it's just a check to be sure that the repository_method is set all the time and not loosed somewhere in the process

Reply:
yes sorry, I made the changes to use `find` if none provided